### PR TITLE
SUBMARINE-292. Merge workbench server into submarine server

### PR DESCRIPTION
**Pull request by liuxunorg**
### What is this PR for?
Now submarine has 2 services, submarine server and workbench server. These two services have no conflicts.
Originally divided into two services, it is for the development of sub-modules, easy to develop.
But this way
1. Users need to maintain two service processes, which increases the cost of use and maintenance of users and administrators.
2. Because we need to be able to run the service on multiple resource scheduling systems(YARN/K8s/Docker), the two services create the complexity of the launcher module.
So, now the service in the workbench server is migrated to the submarine server. But the code for the workbench server remains the same.

### What type of PR is it?
Refactoring

### Todos
* [ ] [SUBMARINE-293](https://issues.apache.org/jira/browse/SUBMARINE-293) - Add submarine server rest test case

### What is the Jira issue?
* https://issues.apache.org/jira/browse/SUBMARINE-292

### How should this be tested?
* [CI Pass](https://travis-ci.org/liuxunorg/submarine/builds/613280558)

### Screenshots (if appropriate)
```
|____bin
| |____common.sh
| |____submarine-daemon.sh
| |____submarine-env.sh
|____commons-runtime-0.3.0-SNAPSHOT.jar
|____conf
| |____log4j.properties
| |____log4j.properties.template
| |____submarine-env.sh.template
| |____submarine-site.xml
| |____submarine-site.xml.template
|____lib
| |____bcpg-jdk15on-1.61.jar
| |____bcpkix-jdk15on-1.61.jar
| |____hk2-utils-2.5.0-b42.jar
| |____interpreter
| | |____python
| | | |____python-interpreter-0.3.0-SNAPSHOT-shade.jar
| | |____spark
| | | |____scala-2.11
| | | | |____spark-scala-2.11-0.9.0-SNAPSHOT.jar
| | | |____spark-interpreter-0.3.0-SNAPSHOT-shade.jar
| |____j2objc-annotations-1.1.jar
| |____jackson-annotations-2.9.10.jar
| |____websocket-server-9.4.18.v20190429.jar
| |____workbench
| | |____workbench-web.war
|____LICENSE
|____licenses
| |____LICENSE-asm.txt
| |____LICENSE-hamcrest.txt
| |____LICENSE-slf4j.txt
| |____LICENSE-tony.txt
|____logs
| |____submarine.log
|____NOTICE
|____server
|____submarine-all-0.3.0-SNAPSHOT-hadoop-2.9.jar
|____submarine-client-0.3.0-SNAPSHOT.jar
|____submitter-yarn-0.3.0-SNAPSHOT.jar
|____tony-core-0.3.21.jar
```

### Questions:
* Does the licenses files need update? Yes/No
* Is there breaking changes for older versions? Yes/No
* Does this needs documentation? Yes/No

### CODE REVIEW

File: submarine-server/server-core/src/test/java/org/apache/submarine/jobserver/rest/api/JobApiTest.java [hunk 1]
```diff
@@ -17,111 +17,104 @@
...
-import javax.ws.rs.core.Response;
-
-import static org.junit.Assert.assertEquals;
+package org.apache.submarine.jobserver.rest.api;
 
-public class JobApiTest extends JerseyTest {
+import org.junit.Test;
 
+public class JobApiTest { //extends JerseyTest {
+  /*
```
Comment (yuanzac) [inline submarine-server/server-core/src/test/java/org/apache/submarine/jobserver/rest/api/JobApiTest.java]: Why do we need to comment on this test class? The test cases are not running after this modification

File: submarine-server/server-submitter/submitter-k8s/pom.xml [hunk 2]
```diff
@@ -30,7 +30,7 @@
 
   <artifactId>submitter-k8s</artifactId>
   <version>0.3.0-SNAPSHOT</version>
-  <name>Submarine: Kubernetes Submitter</name>
+  <name>Submarine: Submitter Kubernetes</name>
```
Comment (yuanzac) [inline submarine-server/server-submitter/submitter-k8s/pom.xml]: Is it better to use Kubernetes Submitter?

File: submarine-server/server-submitter/submitter-yarn/pom.xml [hunk 3]
```diff
@@ -26,7 +26,7 @@
 
   <artifactId>submitter-yarn</artifactId>
   <version>0.3.0-SNAPSHOT</version>
-  <name>Submarine: Server Submitter for YARN</name>
+  <name>Submarine: Submitter YARN</name>
```
Comment (yuanzac) [inline submarine-server/server-submitter/submitter-yarn/pom.xml]: Is it better to use Yarn Submitter?

File: submarine-server/server-submitter/submitter-yarnservice/pom.xml [hunk 4]
```diff
@@ -30,7 +30,7 @@
 
   <artifactId>submitter-yarnservice</artifactId>
   <version>0.3.0-SNAPSHOT</version>
-  <name>Submarine: Server Submitter for YARN Service</name>
+  <name>Submarine: Submitter YARN Service</name>
```
Comment (yuanzac) [inline submarine-server/server-submitter/submitter-yarnservice/pom.xml]: Is it better to use Yarn Service Submitter?

File: submarine-server/server-core/src/test/java/org/apache/submarine/jobserver/rest/api/JobApiTest.java (hunk 1 above)
Comment (liuxunorg) [reply submarine-server/server-core/src/test/java/org/apache/submarine/jobserver/rest/api/JobApiTest.java]: The test cases here have no practical meaning.
I created new JIRA [SUBMARINE-293. Add submarine server rest test case](https://issues.apache.org/jira/browse/SUBMARINE-293)

**Comment by liuxunorg** [comment]
@yuanzac , Thank you help me review, I fixed there issue on your suggestion.

**Comment by yuanzac** [comment]
Thanks, @liuxunorg for your contribution.

**Pull request closed without merging**